modules/react/button/stories/button/examples/PrimaryInverse.tsx

```diff
@@ -28,3 +28,5 @@ export const PrimaryInverse = () => (
     <PrimaryButton icon={relatedActionsVerticalIcon} variant="inverse" />
   </Flex>
 );
+
+export default PrimaryInverse;
```

Comment:
The `export default` rewrite can be done via a RegEx. This is already done in https://github.com/Workday/canvas-kit/blob/master/.storybook/whole-source-loader.js

The exports are extracted using https://github.com/Workday/canvas-kit/blob/master/modules/docs/webpack/extract-exports.js